nncf/quantization/passes.py

```diff
@@ -20,38 +20,18 @@
 TModel = TypeVar("TModel")
 
 
-def transform_to_inference_graph(
+def remove_shapeof_subgraphs_inplace(
     nncf_graph: NNCFGraph,
     shapeof_metatypes: List[OperatorMetatype],
     read_variable_metatypes: Optional[List[OperatorMetatype]] = None,
-) -> NNCFGraph:
+) -> None:
```

Comment:
Since we drop the `transform_to_inference_graph` method, it may be more useful to add a backend-specific method that does the same, but to the `model_utils`, for example, since this method may be useful not only for `MinMax`.

Reply:
I returned `transform_to_inference_graph` function, hope it's better now